spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -473,33 +473,40 @@
           visit_bulk_order_management
         end
 
-        xit "allows filters to be used in combination" do
-          pending "#9811"
+        it "allows filters to be used in combination" do
           expect(page).to have_selector "tr#li_#{li1.id}"
           expect(page).to have_selector "tr#li_#{li2.id}"
-          select2_select oc1.name, from: "order_cycle_filter"
+          find("#s2id_order_cycle_filter").click
+          find(".select2-result-label", text: oc1.name.to_s).click
```

Comment:
This is now repeated a lot. We should introduce a better helper for this. Maybe like this?
```suggestion
          click_on_select2("order_cycle_filter", oc1.name)
```

Reply:
Done on 3447959a061eb1d3867eee1e5f55556dcdfcfb63 :+1: